base/src/main/java/com/smartdevicelink/proxy/rpc/SubscribeVehicleDataResponse.java

```diff
@@ -239,20 +248,28 @@ public VehicleDataResult getInstantFuelConsumption() {
     }
 
     /**
-     * Sets External Temperature
+     * Sets the externalTemperature.
      *
-     * @param externalTemperature a VehicleDataResult related to external temperature
+     * @param externalTemperature The external temperature in degrees celsius. This parameter is deprecated starting RPC
+     * Spec 7.1, please see climateData.
+     * @since SmartDeviceLink 2.0.0
+     * @deprecated in SmartDeviceLink 7.1.0
      */
+    @Deprecated
     public SubscribeVehicleDataResponse setExternalTemperature(VehicleDataResult externalTemperature) {
         setParameters(KEY_EXTERNAL_TEMPERATURE, externalTemperature);
         return this;
     }
 
     /**
-     * Gets External Temperature
+     * Gets the externalTemperature.
      *
-     * @return a VehicleDataResult related to external temperature
+     * @return VehicleDataResult The external temperature in degrees celsius. This parameter is deprecated starting RPC
+     * Spec 7.1, please see climateData.

```

Comment:
@kostyaBoss When using the RPC generator tool, on line 268 I got 7.1.0.  I would replace 7.1 with 7.1.0.